traffic_portal/app/src/common/api/ProfileParameterService.js

```diff
@@ -35,6 +35,7 @@ var ProfileParameterService = function($http, messageModel, ENV) {
 	this.linkProfileParameters = function(profile, params) {
 		return $http.post(ENV.api['root'] + 'profileparameter', { profileId: profile.id, paramIds: params, replace: true }).then(
 			function(result) {
+				messageModel.setMessages([ { level: 'success', text: 'Parameters linked to profile' } ], false);
```

Comment:
i dont think this is supposed to be in here as every call to linkProfileParameters takes care of the messaging.

plus, obviously not restangular related

Reply:
I actually couldn't remember why I did that. Thanks for catching that.

Reply:
yep, no worries. remove that and i'll merge this.